en/getting_started/rc_transmitter_receiver.md

```diff
@@ -97,6 +97,7 @@ In addition to the transmitter/receiver pairs being compatible, the receiver mus
 As general guidance, receivers connect to the flight controller using the port appropriate to their supported protocol:
 
 - Spektrum and DSM receivers must connect to a **SPKT/DSM** input.
+- Graupner receiver SUMD ouput must connect to a **SPKT/DSM** input.

```

Comment:
Typo "ouput".

@Benoit3 Thank you. Can you please confirm how this works? I Think you saying that:
- the Graupner receivers support SUMD output, and that PX4 can read this if you connect to the SPKT/GSM input. Is that correct?
- Is it also correct you can't connect to other inputs (e.g. RC In)
- Does this work for any of the Graupner receivers, or is it some subset?

Reply:
Hi @Benoit3 - did you see my questions ^^^